tests/decorators/test_security.py

```diff
@@ -165,3 +165,46 @@ def apikey_info(apikey, required_scopes=None):
     request.headers = {"X-Auth": 'foobar'}
 
     assert wrapped_func(request, ['admin']) is not None
+
+
+def test_multiple_schemes(security_handler_factory):
```

Comment:
Does it make sense to add a test for two different schemes as well?

Reply:
Not sure, could do it indeed. 
I believe that if the other individual security functions are unit tested, then this test should also pass because it's mainly testing the `verify_multiple_schemes` function.

Reply:
Having looked a bit more into the code, I agree that this should be sufficient.